device.go

```diff
@@ -170,11 +174,66 @@ func newBlockDevice(devInfo DeviceInfo) *BlockDevice {
 	}
 }
 
+func makeBlockDevIDForHypervisor(deviceID string) string {
+	return fmt.Sprintf("drive-%s", deviceID)
```

Comment:
@amshinde Please do the truncating as part of this routine.

Reply:
Done